sys/suit/transport/vfs.c

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (C) 2022 ML!PA Consulting GmbH
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     sys_suit_transport_vfs
+ * @{
+ *
+ * @fil
+ * @brief       SUIT VFS
+ *
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+ * @}
+ */
+
+#include <fcntl.h>
+#include <string.h>
+
+#include "suit/transport/vfs.h"
+#include "vfs.h"
+
+#define ENABLE_DEBUG 0
+#include "debug.h"
```

Comment:
Seems like `transport/coap` was using `log.h` instead of `debug.h`, could we use the same everywhere?